src/cache.ts

```diff
@@ -301,6 +301,24 @@ class Lock {
 
   constructor({ cache = getDefaultCachePath() } = {}) {
     this.path = resolve(cache, ".lock");
+    this.checkExpiredLockPath();
+  }
+  
+  /** Clean expired lock path */
+  checkExpiredLockPath() {
+    // if this.path's create time is older than cacheTTL, remove it
+    try {
+      const fileInfo = Deno.statSync(this.path);
+      const lockTTL = 300 * 1000;
+      if (fileInfo.birthtime && Date.now() - fileInfo.birthtime.getTime() < lockTTL) {
+        // Deno.removeSync(this.path);
+        console.warn("%c There is an old lock file(%s), you should remove it.", 'color: #ff0000', this.path);
+      }
```

Comment:
I wonder if we should just remove it. If not, this behavior is fine in CI but not fine for a regular user to encounter. If we're not in CI we should prompt the user with "there is an old lock file, this is probably due to a failed download, do you want to remove it? [Y/n]" or something like that

Reply:
I prefer to just remove it. Simple for user and fine for CI.
```
Deno.removeSync(this.path);
console.log(`%c There is an old lock file (${this.path}), this is probably due to a failed download. It has been removed automatically.`, 'color: #ff0000');
```

Prompt is not fine for CI as you said.

        const userInput = prompt(`\x1b[31m There is an old lock file (${this.path}), this is probably due to a failed download. Do you want to remove it? [Y/n]:\x1b[0m`);
        if (userInput && userInput.toLowerCase() === 'y') {
          Deno.removeSync(this.path);
          console.log(`Old lock file ${this.path} has been removed.`);
        } else {
          console.log(`Lock file ${this.path} has not been removed.`);
        }